generator/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+
+junit_family  = legacy
```

Comment:
I didn't know about pytest initialization files... Thanks.

If we set up Jenkins it looks like the `xunit2` option might be nice.